travis/push-to-cicd-status.sh

```diff
@@ -55,7 +59,13 @@ add_acc_provision_artifacts() {
 }
 
 update_acc_provision_release(){
-    python $SCRIPTS_DIR/update-release.py "${PYPI_REGISTRY}" "${TAG_NAME}"
+    DIR="/tmp/${GIT_LOCAL_DIR}/docs/release_artifacts/${RELEASE_TAG}"
+    if [[ "${RELEASE_TAG}" =~ *rc ]] && [[ ${RELEASE} == "false" ]] ; then
+        DIR="/tmp/${GIT_LOCAL_DIR}/docs/release_artifacts/${RELEASE_Z_TAG}"
```

Comment:
you are missing "${TRAVIS_REPO_SLUG##*/} in the last for DIR

Reply:
thank you